c/pthread-memsafety/list2_false-valid-deref.c

```diff
@@ -43,8 +43,8 @@ int main(int argc, char **argv) {
   pthread_t id1, id2;
   struct item *list = ((void *)0);
   pthread_create(&id1, NULL, build, list);
-  pthread_create(&id2, NULL, delete, list);
   pthread_join(id1, NULL);
+  pthread_create(&id2, NULL, delete, list);
```

Comment:
This change removes the main part of the concurrent program.
The program should have concurrent access to a list and then dereference some already deleted element.